src/main/java/org/mvplugins/multiverse/core/world/WorldConfigNodes.java

```diff
@@ -118,7 +116,33 @@ private <T> ConfigNode<T> node(ConfigNode.Builder<T, ?> nodeBuilder) {
     final ConfigNode<Biome> BIOME = node(ConfigNode.builder("biome", Biome.class)
             .defaultValue(Biome.CUSTOM)
             .name(null)
-            .serializer(new BiomeSerializer()));
```

Comment:
Why not keep these classes?

Reply:
it's one time use, I don't think it needs to be separated